### PR TITLE
feat: adds new flags to tootctl media remove

### PR DESCRIPTION
**Pull request by quicoto**
Adds documentation from this PR https://github.com/mastodon/mastodon/pull/22149/files landing in the next Mastodon stable 4.1.0

### CODE REVIEW

**Comment by quicoto** [comment]
@marians @evanphilip 👋

File: content/en/admin/tootctl.md [hunk 1]
```diff
@@ -644,14 +644,23 @@ Fix corrupted database indexes that may have been caused due to changing collati
...
+: How old media attachments have to be before they are removed. In case of avatars and headers, how old the last webfinger request and update to the user has to be before they are removed. Defaults to 7.
 
 `--concurrency N`
 : The number of workers to use for this task. Defaults to N=5.
 
+`--prune-profiles`
+: Instead of media attachments, remove locally cached copies of avatars and headers from other servers. Cannot be combined with --remove-headers.
+
+`--remove-headers`
+: Instead of media attachments, remove locally cached copies of headers from other servers. Cannot be combined with --prune-profiles.
```
Comment (evanphilip) [inline content/en/admin/tootctl.md:659]: backticks?

**Review by evanphilip — APPROVED** [review]

**Comment by quicoto** [comment]
@Gargron can you review this one?

Thank you

**Comment by evanphilip** [comment]
We are reworking the flags of `tootctl media remove` in https://github.com/mastodon/mastodon/issues/23628. 

We don't know if/when that will get approved, so this is certainly not a suggestion to withdraw this update to the documentation.
Also, this time, am attaching an update to the documentation to the same issue to prevent outdated documentation like last time.

**Comment by ClearlyClaire** [comment]
> We are reworking the flags of `tootctl media remove` in [mastodon/mastodon#23628](https://github.com/mastodon/mastodon/issues/23628).
> 
> We don't know if/when that will get approved, so this is certainly not a suggestion to withdraw this update to the documentation. Also, this time, am attaching an update to the documentation to the same issue to prevent outdated documentation like last time.

I think it makes sense including, since at least one release has those flags. It would make sense to better highlight when those options were added but I'm not sure how to do that.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**